.github/workflows/cuda-gpu.yml

```diff
@@ -0,0 +1,37 @@
+---
+name: 🔥 CUDA GPU
```

Comment:
This workflow seems like it is just a duplicate of the unit test job in the ci-weekly-gpu workflow. Is there something different about it I am overlooking?

Reply:
The `cuda-gpu.yml` was replaced by `ci-weekly-gpu.yml` and has been deleted.